app/assets/javascripts/discourse/app/lib/utilities.js

```diff
@@ -41,7 +41,7 @@ export function escapeExpression(string) {
   }
 
   // don't escape SafeStrings, since they're already safe
-  if (string instanceof Handlebars.SafeString) {
+  if (Handlebars.SafeString && string instanceof Handlebars.SafeString) {
```

Comment:
@CvX @davidtaylorhq I don't think `Handlebars.SafeString` is defined in the context of MiniRacer, that would fail in this case